javarosa/src/main/java/org/javarosa/core/services/locale/Localizer.java

```diff
@@ -47,17 +44,14 @@ public Localizer() {
     }
 
     /**
-     * Full constructor.
-     *
      * @param fallbackDefaultLocale If true, search the default locale when no translation for a particular text handle
      *                              is found in the current locale.
      * @param fallbackDefaultForm   If true, search the default text form when no translation is available for the
      *                              specified text form ('long', 'short', etc.). Note: form is specified by appending ';[form]' onto the text ID.
      */
     public Localizer(boolean fallbackDefaultLocale, boolean fallbackDefaultForm) {
-        stringTree = new PrefixTree(10);
-        localeResources = new OrderedHashtable<>();

```

Comment:
Is this (Ordered -> Non Ordered) vital for performance? I don't remember why these are ordered, but it seems like an odd detail to exist and may have relevance.